sql/src/test/scala/org/apache/spark/sql/sedona_sql/UDT/GeometryUdtTestScala.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you please move this test file to the path "org/apache/sedona/sql"?

Reply:
Sure, I thought to put it in the same package as the GeometryUDT class.